BUILD

```diff
@@ -1 +1,9 @@
 # Empty Bazel build file
+java_binary(
+    name = "ClosureCommandLineCompiler",
```

Comment:
This doesn't make sense as a top level target - there were no top level target before. Also the name is very misleading and also the build file still comments above as "empty"